app/javascript/components/physical-storage-form/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
```

Comment:
Can you please split the external imports from the internal ones with an empty line?